templates/deployment-back.yaml

```diff
@@ -121,6 +121,8 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
             {{- end }}
+            - name: CORS_ALLOWED_ORIGIN_PATTERN
+              value: {{ default "https?://.*.neotys.com.*" (printf "%s%s" (include "nlweb.helpers.getScheme" .) (.Values.services.files.host) | quote) }}
```

Comment:
Should be based on Values.services.webapp.host ?

Reply:
Yes